From a7ff696e94d22542fcf4511883dc516a95ae74cc Mon Sep 17 00:00:00 2001
From: Alex Evraev <alexev@marvell.com>
Date: Thu, 20 Jun 2019 17:58:22 +0300
Subject: [PATCH 1177/1200] configs: add eth4 address to def env

Eth4 address is not defined at default environment.
For CN9132 platform, this causes error message at u-boot stage.

Change-Id: I723e5d9037351b18069a5610cd7e73482dbd0e60
Signed-off-by: Alex Evraev <alexev@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/11353
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Reviewed-by: Kostya Porotchkin <Kostya.Porotchkin@cavium.com>
---
 include/configs/mvebu_armada-common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/mvebu_armada-common.h b/include/configs/mvebu_armada-common.h
index 382fae74fe..5078eba28d 100644
--- a/include/configs/mvebu_armada-common.h
+++ b/include/configs/mvebu_armada-common.h
@@ -56,6 +56,7 @@
 					"eth1addr=00:51:82:11:22:01\0"	\
 					"eth2addr=00:51:82:11:22:02\0"	\
 					"eth3addr=00:51:82:11:22:03\0"	\
+					"eth4addr=00:51:82:11:22:04\0"	\
 					"image_name=Image\0"		\
 					"get_ramfs=if test \"${ramfs_name}\"" \
 						" != \"-\"; then setenv " \
-- 
2.22.0

